infoschema/tables_test.go

```diff
@@ -1552,3 +1562,51 @@ func (s *testTableSuite) TestInfoschemaDeadlockPrivilege(c *C) {
 	}, nil, nil), IsTrue)
 	_ = tk.MustQuery("select * from information_schema.deadlocks")
 }
+
+func (s *testDataLockWaitSuite) SetUpSuite(c *C) {
+	testleak.BeforeTest()
+
+	client, pdClient, cluster, err := unistore.New("")
```

Comment:
I think the more proper way to write the unit tests is to implement the GetLockWaitInfo api for the unistore, and construct a real lock waiting here. Since finishing that kind of tests would be too complicated and we may not have so much time, it's ok to me that we do not do that in this PR. But I think the tests need to be improved in the future.

Reply:
I think there should be a unit test on unistore for testing GetLockWaitInfo api. (Will implement in the future)
And an itegration test for the whole DATA_LOCK_WAIT stuff for the whole system of TiDB-TiKV/unistore.(But I'm not very sure where should I put that test in.)

Reply:
You can see my tests for the deadlocks table, perhaps it may help you. If some integration tests are hard to implement in TiDB repo, or you relies on some specific configuration of the cluster, you can do it in UTF.

Reply:
Can you reuse the `SetUpSuite` and `TearDownSuite` of the `testTableSuiteBase`, which means, invoking  `s.testTableSuiteBase.SetUpSuite` and only do additional work in your own code?

Reply:
SetUpSuite is very different with testTableSuiteBase, but we can just use testTableSuiteBase's TearDownSuite here. So no need to implement TearDownSuite.

Reply:
If they are totally different, I think it's better to not nesting the testTableSuiteBase here, but writing the similar fields by yourself.  But let's merge the PR first and you can do code refinements later.